guardian/manage-frontend
# Copy changes to Switching Journey

## What does this change?
This PR has some copy changes to the switching journey:

- Small print wording on Switch Options for both above and below threshold on any journey
- Switch Options heading wording amend for above and below threshold when on the app
- Switch Review wording amended for both above and below threshold on any journey



## How to test
Check screens in CODE or locally via Storybook (where the below screenshots have been taken) and check that the wording is correct and displaying as expected.

## Images
**App**
Switch Option Above Threshold on app
![image](https://user-images.githubusercontent.com/115992455/225880011-8634ecd6-7592-4f50-9669-531edb37e7f1.png)

Switch Option Below Threshold on app
![image](https://user-images.githubusercontent.com/115992455/224731157-5607591e-8679-4742-8edc-19dcf75a054f.png)

Switch Review above threshold on app
![image](https://user-images.githubusercontent.com/115992455/224731947-78cd990a-e93b-45e9-86e9-6362a8127f8e.png)

Switch Review below threshold on app
![image](https://user-images.githubusercontent.com/115992455/224732224-f8c35725-f242-4f92-b5b1-a127bdcf3c96.png)

-----
**Non-App**
Switch Option Above Threshold non app
![image](https://user-images.githubusercontent.com/115992455/224735014-93677937-bafa-4da4-8825-d785e4575bc5.png)

Switch Option Below Threshold non app
![image](https://user-images.githubusercontent.com/115992455/224733767-93f9b20e-a970-4498-a317-bf1f73c06ebd.png)

Switch Review above threshold non app
![image](https://user-images.githubusercontent.com/115992455/224734068-71cb4977-5abf-4469-952d-6508188ecccf.png)

Switch Review below threshold non app
![image](https://user-images.githubusercontent.com/115992455/224734213-4842f33c-8e1f-47a0-8af5-a3abc234c413.png)



## Accessibility
